pennylane/ops/qubit/parametric_ops_multi_qubit.py

```diff
@@ -595,6 +595,111 @@ def _pauli_rot_resources(pauli_word):
 add_decomps(PauliRot, pauli_rot_decomposition)
 
 
+def _decomp_int_to_powers_of_two(k: int, n: int) -> list[int]:
+    r"""Decompose an integer :math:`k<=2^{n-1}` into additions and subtractions of the
+    smallest-possible number of powers of two.
+
+    Args:
+        k (int): Integer to be decomposed
+        n (int): Number of bits to consider
+
+    Returns:
+        list[int]: A list with length ``n``, with entry :math:`c_i` at position :math:`i`.
+
+    This function follows the logic described in the documentation of
+    PCPhase.compute_decomposition.
```

Comment:
I think a quick example could help get the point across of this function, both here and in the description in PCPhase, e.g. the one from your comment in the tests:
The "naive" decomposition of $k=121=01111001_2$ in powers of 2 is $1 2^6 + 1 2^5 + 1 2^4 + 1 2^3 + 1 2^0$ (just reading off the 1s). But we can do better by writing it as $121 = 128 - 8 + 1$ yielding only 3 non-zero entries.

Reply:
Good call! I added it to the (unrendered) doc string of `_decomp_int_to_powers_of_two` and as a second calculation example to the end of the `PCPhase.compute_decomposition` docstring. Is this what you had in mind? :)